code-challenges.test.js

```diff
@@ -46,6 +91,38 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
+//the words in the output arent all capatalized and it took my a while to realize and i kept receiving an error.
+
+describe(`capWords`, () => {
+    it(`returns an array with all the words capitalized`, () => {
+        const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+        const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+        expect(capWords(randomNouns1)).toEqual(["STREETLAMP", "POTATO", "TEETH", "CONCLUSION", "NEPHEW"])
+        expect(capWords(randomNouns2)).toEqual(["TEMPERATURE", "DATABASE", "CHOPSTICKS", "MANGO"])
+    })
+})
```

Comment:
Pay attention to the expected outputs for each test variable. The words should be capitalized only at the first letter of each word, not the whole word itself.